CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - `[jest-environment-jsdom]` [**BREAKING**] Upgrade JSDOM to v22 ([#13825](https://github.com/jestjs/jest/pull/13825))
 - `[@jest/environment-jsdom-abstract]` Introduce new package which abstracts over the `jsdom` environment, allowing usage of custom versions of JSDOM ([#14717](https://github.com/jestjs/jest/pull/14717))
 - `[jest-environment-node]` Update jest environment with dispose symbols `Symbol` ([#14888](https://github.com/jestjs/jest/pull/14888) & [#14909](https://github.com/jestjs/jest/pull/14909))
+- `[expect, @jest/expect]` [**BREAKING**] Add type inference for function parameters in `CalledWith` assertions ([#15129](https://github.com/facebook/jest/pull/15129))
```

Comment:
why is this a breaking change? the odl type was just `Function` or something?

also, only `expect` seems to be impacted? I don't see any changes to  `@jest/expect` at least 😀

Reply:
@SimenB I'm thinking it will likely break some existing tests that use "have been called" functions due to incorrect typing, like it did in tests/examples in this repo. I'm not sure what are the full implications of this "tag", so obviously I will let you decide.

As for `@jest/expect`, I merely copied it from the last PR that tried to introduce this feature. I can obviously remove it.

Reply:
It's just used as a way to catch the reader's eye.

I really should have included a migration guide thing when I started landing breaking changes, where _how_ the breaking change might affect you would be listed 😅 I'll have to add one